Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -79,24 +84,44 @@ STPPaymentCardTextFieldDelegate, MessageBannerViewControllerPresenting {
       |> \.returnKeyType .~ .next
       |> \.textAlignment .~ .right
       |> \.textColor .~ .ksr_text_dark_grey_500
+
+    _ = self.cardholderNameTextField
+      |> \.accessibilityLabel .~ self.cardholderNameLabel.text
       |> \.attributedPlaceholder .~ NSAttributedString(
           string: Strings.Name(),
           attributes: [NSAttributedString.Key.foregroundColor: UIColor.ksr_text_dark_grey_400])
 
     _ = self.creditCardTextField
       |> \.borderColor .~ nil
       |> \.font .~ .ksr_body()
+
+    _ = self.creditCardTextField
       |> \.cursorColor .~ .ksr_green_700
       |> \.textColor .~ .ksr_text_dark_grey_500
       |> \.placeholderColor .~ .ksr_text_dark_grey_400
+
+    _ = self.creditCardValidationErrorLabel
+      |> settingsDescriptionLabelStyle
+      |> \.textColor .~ .ksr_red_400
+      |> \.text %~ { _ in Strings.Unsupported_card_type() }
   }
 
   override func bindViewModel() {
     super.bindViewModel()
 
-     self.cardholderNameTextField.rac.becomeFirstResponder =
+    self.creditCardValidationErrorContainer.rac.hidden =
+      self.viewModel.outputs.cardBrandUnsupportedErrorMessageHidden
+    self.cardholderNameTextField.rac.becomeFirstResponder =
       self.viewModel.outputs.cardholderNameBecomeFirstResponder
 
+    self.viewModel.outputs.cardBrandUnsupportedErrorMessageHidden
+      .filter { isFalse($0) }
+      .observeForUI()
+      .observeValues { _ in
+        UIAccessibility.post(notification: .layoutChanged,
```

Comment:
This is amazing!!!

I had no idea the keyboard will stay on and this will briefly interrupt the typing.

![giphy](https://media.giphy.com/media/odoDS1U6v9ldu/giphy.gif)

Reply:
Cc @cdolm92